tests/test_logger.py

```diff
@@ -36,18 +36,19 @@ def test_logger_exists(fs, monkeypatch):
     }
 
     fs.create_dir("/dev")
+    fs.create_dir("foo")
```

Comment:
If it's really a pain to debug and make it works, you could try using `tmp_path` instead (cf. https://docs.pytest.org/en/7.1.x/how-to/tmp_path.html )